README.md

```diff
@@ -1,3 +1,83 @@
 # HyP3 Event Monitoring
 
-A software stack to allow users to enable event monitoring and processing of data as it becomes available.
\ No newline at end of file
+A software stack that allows automatic submission of jobs to hyp3 over a specified area to easily monitor areas of interest.
+
+## Table of contents
+- [Deployment](#deployment)
+  - [Prerequisites](#prerequisites)
+  - [Stack Parameters](#stack-parameters)
+  - [Deploy with CloudFormation](#deploy-with-cloudformation)
+- [Testing](#testing)
+
+## Deployment
+
+### Prerequisites
+These resources are required for a successful deployment, but managed separately:
+
+- HyP3 target deplotment
```

Comment:
`deployment` typo

Reply:
would also be nice to have an example hyp3 url somewhere, e.g. `https://hyp3-api.asf.alaska.edu`